src/pages/landing/deriv-go/_banner.js

```diff
@@ -0,0 +1,221 @@
+const TopContent = styled(Flex)`
+    max-width: 996px;
+    padding: 0;
+    margin: 0 auto;
+    justify-content: flex-start;
+
+    @media ${device.laptopM} {
+        padding: 0 40px;
+    }
+    @media ${device.tabletL} {
+        max-width: 58.8rem;
+        flex-direction: column;
+        justify-content: center;
+        height: 367px;
+        padding: 0 16px;
+    }
+`
+const TopContentLeft = styled(Flex)`
+    flex-direction: column;
+    width: 486px;
+    margin-right: 126px;
+    justify-content: flex-start;
+    align-items: flex-start;
+    @media ${device.laptopM} {
+        margin-right: 80px;
+    }
+    @media ${device.tabletL} {
+        justify-content: center;
+        margin-right: 0;
+        width: 100%;
+    }
+`
+const TopContentLogo = styled.img`
+    width: 213px;
+    height: 28px;
+    margin-top: 32px;
+    margin-bottom: 71px;
+    @media ${device.tabletL} {
+        width: 190px;
+        height: 25px;
+        margin: 32px auto 39px;
+    }
+`
+const StyledHeader = styled(Header)`
+    margin-bottom: 16px;
+    line-height: 1.25;
+    @media ${device.tabletL} {
+        font-size: 40px;
+        text-align: center;
+    }
+`
+const StyledHeaderSmall = styled(Header)`
+    max-width: 58rem;
+    line-height: 1.5;
+    margin-bottom: 40px;
+    @media ${device.tabletL} {
+        font-size: 20px;
+        margin-bottom: 24px;
+        max-width: 100%;
+        text-align: center;
+    }
+`
+const TopAppStoreWrapper = styled(Flex)`
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    height: 40px;
+    @media ${device.tabletL} {
+        flex-direction: column;
+        justify-content: center;
+        height: unset;
+    }
+`
+const GoogleStoreLogo = styled.img`
+    width: 138px;
+    height: 40px;
+    margin-right: 8px;
+    @media ${device.tabletL} {
+        margin-right: unset;
+        margin-bottom: 6px;
+    }
+`
+const StyledSmallAppstore = styled(Header)`
+    max-width: 58rem;
+    line-height: 1.5;
+    @media ${device.tabletL} {
+        max-width: 100%;
+        text-align: center;
+    }
+`
+const BottomWrapper = styled(Flex)`
+    max-width: 996px;
+    height: 126px;
+    padding: 0;
+    margin: 0 auto;
+    justify-content: flex-end;
+    position: relative;
+
+    @media ${device.tabletL} {
+        max-width: 58.8rem;
+        height: 313px;
+        justify-content: center;
+        padding: 0 16px;
+    }
+`
+const BannerImageWrapper = styled(Flex)`
+    width: 384px;
+    display: flex;
+    align-items: flex-end;
+    position: absolute;
+    bottom: 0;
+    @media ${device.tabletL} {
+        max-width: 288px;
+        width: 100%;
+        bottom: 56px;
+    }
+`
+const DesktopImageWrapper = styled(Container)`
+    width: 100%;
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const MobileImageWrapper = styled(Container)`
+    display: none;
+    @media ${device.tabletL} {
+        display: flex;
+        width: 100%;
+    }
+`
```

Comment:
If your intention is just for having either a desktop/mobile view. A suggestion would to extend the styles from either `div` or `Flex` instead of `Container` as the container will have some widths set across different breakpoints that may be unintended.

Reply:
Amended in the latest commit. Changed to div.